docs/cloud/manage/role-based-access.md

```diff
@@ -0,0 +1,139 @@
+<!--
+title: "Role-Based Access model"
+sidebar_label: "Role-Based Access model"
+custom_edit_url: "https://github.com/netdata/netdata/blob/master/docs/cloud/manage/role-based-access-model.md)"
+sidebar_position: "1"
+learn_status: "Published"
+learn_topic_type: "Tasks"
```

Comment:
Just a quick question:

This document clearly present the RBA of Netdata cloud but you place it under `learn_rel_path: "Concepts"`, but you categorize it as task? `learn_topic_type: "Tasks"`

Reply:
oh, good catch @tkatsoulas I think it was a copy+paste I forgot to review. will change it